lib/KTable/KTable.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+
+  <div class="k-table">
+    <table>
+      <caption v-if="caption">
+        {{ caption }}
+      </caption>
+      <thead>
+        <tr>
+          <th
+            v-for="(header, index) in headers"
+            :key="index"
+            tabindex="0"
+            @click="handleSort(index)"
+            @keydown="handleKeydown($event, -1, index)"
+          >
+            {{ header.label }}
+            <span v-if="header.dataType !== 'others'" class="sort-icon">
+              <span v-if="sortKey === index && sortOrder === 'asc'">⬆️</span>
+              <span v-else-if="sortKey === index && sortOrder === 'desc'">⬇️</span>
+              <span v-else>⬍</span>
+            </span>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(row, rowIndex) in sortedRows" :key="rowIndex">
+          <KTableGridItem
+            v-for="(cell, cellIndex) in row"
+            :key="cellIndex"
+            :content="cell"
+            :align="headers[cellIndex].dataType === 'numeric' ? 'right' : 'left'"
+            :rowIndex="rowIndex"
+            :cellIndex="cellIndex"
+            @keydown="handleKeydown($event, rowIndex, cellIndex)"
+          />
+        </tr>
+      </tbody>
+    </table>
+  </div>
+
+</template>
+
+
+<script>
+
+  import KTableGridItem from './KTableGridItem.vue';
+
+  export default {
+    name: 'KTable',
+    components: {
+      KTableGridItem,
+    },
+    props: {
+      headers: {
```

Comment:
No need to do it now, rather a note for when we're finalizing API (that is shortly before using the table in Kolibri) - I recommend documenting props, slots, and events with JSDocs comments.

Some examples
- https://github.com/learningequality/kolibri-design-system/blob/release-v4/lib/KImg/index.vue#L177-L201
- https://github.com/learningequality/kolibri-design-system/blob/release-v4/lib/KImg/index.vue#L15
- https://github.com/learningequality/kolibri-design-system/blob/release-v4/lib/KImg/index.vue#L364-L366

The documentation will then use these to auto-generate the bottom section of the documentation page https://design-system.learningequality.org/kimg/#props